### PR TITLE
Fixed #26

### LINKED ISSUE #26: To many files open
What can I do to resolve this errors? (Very quickly nonstop repeating at console screen.)
It happens after some normal uptime.

12:08:03 [ALERT] java.io.FileNotFoundException: /mcpe/nemisys/server.log (Too many open files)
        at java.io.FileOutputStream.open0(Native Method)
        at java.io.FileOutputStream.open(FileOutputStream.java:270)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
        at org.itxtech.nemisys.utils.MainLogger.run(MainLogger.java:172)

12:08:03 [ALERT] java.io.FileNotFoundException: /mcpe/nemisys/server.log (Too many open files)
        at java.io.FileOutputStream.open0(Native Method)
        at java.io.FileOutputStream.open(FileOutputStream.java:270)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
        at org.itxtech.nemisys.utils.MainLogger.run(MainLogger.java:172) 

I set ulimit -n 4096, hope this help.

### PR DESCRIPTION
**Pull request by tarik02**
Fixed "Too many open files". You can see here some info about this: http://stackoverflow.com/questions/4289447/java-too-many-open-files. This requires some testers.

### CODE REVIEW

File: src/main/java/org/itxtech/nemisys/utils/MainLogger.java [hunk 1]
```diff
@@ -172,10 +172,16 @@ public void run() {
...
                 } catch (Exception e) {
                     this.logException(e);
                 }
+				finally {
+					try {
+                        writer.close();
+					} catch (Exception e) {
+						this.logException(e);
+					}
+				}
```
Comment (dktapps) [inline src/main/java/org/itxtech/nemisys/utils/MainLogger.java]: Please correct this formatting.

Comment (tarik02) [reply src/main/java/org/itxtech/nemisys/utils/MainLogger.java]: Done.

**Comment by ishitatsuyuki** [comment]
Well, your linked answer mentions a more graceful solution. Why not use that?

**Comment by tarik02** [comment]
@ishitatsuyuki, because I'm want to log errors from that.

**Comment by ishitatsuyuki** [comment]
To do that in correct way, use `Throwable.getSuppressed`. Basically there shouldn't be errors closing, so beautiful code should be more important here.

**Comment by tarik02** [comment]
Tested: this not helps. Close.

**Pull request closed without merging**